### ISSUE TITLE
New PCs on Org Channels

### ISSUE THREAD

**Issue by rozzingit** [Arx-Game/arxcode#30]
**Is your feature request related to a problem? Please describe.**
When a new OC is created or someone picks up a roster, that PC will only be on (I believe) the Public channel. Players have to figure out the process of checking @clist and using addcom to join channels they want. Which isn't terrible or onerous, but it's a few steps, and some people are really not mused to MU* type code at all.

**Describe the solution you'd like**
So I think it'd be really helpful in these cases if newly-apped characters came in on their various org channels. (They can obviously turn them off if they want.) Orgs tend to be the big hubs for new players getting involved, so it'd be good for them to come "pre-installed" on those channels, as it were, for people to greet them and all that jazz. Helps with the welcome!

**Comment by Zironic** [comment]
Yes I agree. It would be nice if you were on all your org channels by default.

**Comment by krakriksdag** [comment]
And org bboards, if applicable!

**Comment by rozzingit** [comment]
I believe they already start subbed to org bboards.
